javascript/src/index.ts

```diff
@@ -643,7 +643,7 @@ class MessageAttempt {
     endpointId: string,
     options?: MessageAttemptListOptions
   ): Promise<ListResponseMessageAttemptEndpointOut> {
-    return this.api.listAttemptsForEndpointApiV1AppAppIdMsgMsgIdEndpointEndpointIdAttemptGet(
+    return this.api.v1MessageAttemptListByEndpointDeprecated(
```

Comment:
How come we have "deprecated" in the name now? Not a biggie, just curious.

Reply:
Because the API call itself is deprecated, and since we renamed the operation IDs it seemed like a good idea to drive the point home by including "deprecated" in the name, not just in the description.

Reply:
OK.